src/huggingface_hub/repository.py

```diff
@@ -397,7 +461,9 @@ def check_git_versions(self):
             )
         logger.info(git_version + "\n" + lfs_version)
 
-    def clone_from(self, repo_url: str, use_auth_token: Union[bool, str, None] = None):
+    def clone_from(
+        self, repo_url: str, use_auth_token: Union[bool, str, None] = None
+    ) -> bool:
```

Comment:
Maybe I'm missing something, but where/why are we returning a bool here?

Reply:
Very nice catch, that's a mistake from my side